rflx/pyrflx/typevalue.py

```diff
@@ -606,22 +612,74 @@ def __eq__(self, other: object) -> bool:
     def equal_type(self, other: Type) -> bool:
         return self.identifier == other.identifier
 
+    @property
+    def model(self) -> Message:
+        return self._type
+
+    @property
+    def path(self) -> Sequence[Link]:
+        set_fields = [
+            Field(field_name)
+            for field_name, field in self._fields.items()
+            if field_name in self.valid_fields
+        ]
+        set_fields.append(FINAL)
+        set_fields.append(INITIAL)
+
+        path: List[Link] = []
+        for field in set_fields:
+            for outgoing_link in self._type.outgoing(field):
+                if (
+                    outgoing_link.target in set_fields
+                    and self.__simplified(outgoing_link.condition) == TRUE
+                ):
+                    path.append(outgoing_link)
+
+        all_field_names = ["Initial"] + self.fields + ["Final"]
+        path.sort(key=lambda i: all_field_names.index(i.source.name))
+        return path
```

Comment:
Couldn't the sorting be removed when adding `INITIAL` as first element of `set_fields`?

Reply:
This was in recalculation approach. Removed it as we are sticking with the original approach.